R/MappedData.R

```diff
@@ -233,12 +343,12 @@ MappedData <- R6::R6Class(
 
       return(invisible(self))
     },
-    #' adds new columns ymin and ymax if reequired
+    #' adds new columns ymin and ymax if required
     translateErrorAestethics = function() {
       if (!private$aestheticExists(paste(private$direction, "min")) |
         !private$aestheticExists(paste(private$direction, "max"))) {
         errorType <-
-          intersect(names(self$mapping), c("error", "error_relativ"))
+          intersect(names(self$mapping), c("error", "error_relative"))
```

Comment:
we should define constants for those strings (e.g. via the enumeration "ColumnNames" or whatsoever; compare
https://github.com/Open-Systems-Pharmacology/TLF-Library/blob/1a74ca24c2d318b5d1a6dc52f51121e1ca00c6c1/R/utilities-enums.R#L43-L48

Reply:
not necessarily in this PR, but in general